tests/foreman/ui/test_discoveredhost.py

```diff
@@ -84,7 +84,8 @@ def test_positive_provision_pxe_host(
     wait_for(
         lambda: sat.api.DiscoveredHost().search(query={'mac': mac}) != [],
         timeout=1500,
-        delay=20,
+        retries=2,
+        delay=40,
```

Comment:
What function consumes the `retries=2` kwarg?

ref: https://github.com/RedHatQE/wait_for/blob/master/wait_for/__init__.py

Reply:
I checked it, it doesn't take retries. I will update it.